docs/run/run-jobs-in-session.mdx

```diff
@@ -108,11 +108,33 @@ with Session(service=service, backend=backend, max_time="25m"):
   ...   
 ```
 
-There is also an interactive timeout value (ITTL, or interactive time to live) that cannot be configured. For premium accounts, the ITTL is 5 min, and for open plan accounts, the ITTL is 2 sec. If no session jobs are queued within that window, the session is temporarily deactivated. For more details about session length and timeout, see [How long a session stays active](sessions#active).
+There is also an interactive timeout value (ITTL, or interactive time to live) that cannot be configured.  If no session jobs are queued within that window, the session is temporarily deactivated. For more details about session length and timeout, see [How long a session stays active](sessions#active). To determine a session's ITTL, follow the instructions in [Determine session details](#session-details) and look for the `interactive_timeout` value.
 
 <span id="close"></span>
 ## Close a session 
 
+### For `qiskit-ibm-runtime` 0.13 or later releases
+
+When the session context manager is exited, the session is put into `In progress, not accepting new jobs` status.  This means that the session will finish processing all running or queued jobs until the maximum timeout value is reached.  After all jobs are completed, the session is immediately closed. This allows the
+scheduler to run the next job without waiting for the session interactive timeout,
+therefore reducing the average job queueing time. You cannot submit jobs to a
+closed session.
+
+``` python
+with Session(service=service, backend=backend) as session:
+    estimator = Estimator()
+    job = estimator.run(...)
+    
+# The session is no longer accepting jobs but the submitted job will run to completion    
+result = job.result()
+```
+
+<Admonition type="note" title="Important">
+    Previously, `session.close()` **canceled** the session.  Starting with `qiskit-ibm-runtime` 0.13, `session.close()` **closes** the session. 
+</Admonition>
+
+### For Qiskit runtime releases before 0.13
+
```

Comment:
I don't think we need to mention the behavior for pre-0.13 release, since there was no way to "close" a session (based on how this doc describe as "close").